apps/ensadmin/src/components/connection/config-info/config-info.tsx

```diff
@@ -7,61 +7,86 @@
 
 import { ChainIcon } from "@/components/chains/ChainIcon";
 import { ConfigInfoAppCard } from "@/components/connection/config-info/app-card";
-import { CopyButton } from "@/components/copy-button";
-import { ErrorInfo, ErrorInfoProps } from "@/components/error-info";
+import { ErrorInfo } from "@/components/error-info";
 import { HealIcon } from "@/components/icons/HealIcon";
 import { IndexAdditionalRecordsIcon } from "@/components/icons/IndexAdditionalRecordsIcon";
-import { ENSAdminIcon } from "@/components/icons/ensnode-apps/ensadmin-icon";
 import { ENSDbIcon } from "@/components/icons/ensnode-apps/ensdb-icon";
 import { ENSIndexerIcon } from "@/components/icons/ensnode-apps/ensindexer-icon";
 import { ENSNodeIcon } from "@/components/icons/ensnode-apps/ensnode-icon";
 import { ENSRainbowIcon } from "@/components/icons/ensnode-apps/ensrainbow-icon";
 import { IconGraphNetwork } from "@/components/icons/graph-network";
 import { ExternalLinkWithIcon } from "@/components/link";
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Skeleton } from "@/components/ui/skeleton";
 import { Tooltip, TooltipContent, TooltipTrigger } from "@/components/ui/tooltip";
-import { useSelectedConnection } from "@/hooks/active/use-selected-connection";
 import { getChainName } from "@/lib/namespace-utils";
 import { cn } from "@/lib/utils";
+import { useENSIndexerConfig } from "@ensnode/ensnode-react";
 import { ENSIndexerPublicConfig } from "@ensnode/ensnode-sdk";
-import { PlugZap, Replace } from "lucide-react";
+import { Replace } from "lucide-react";
 
 /**
- * ENSNodeConfigInfo display variations:
- *
- * Standard - ensIndexerConfig: ENSIndexerPublicConfig, error: undefined
- * Loading - ensIndexerConfig: undefined, error: undefined
- * Error - ensIndexerConfig: undefined, error: ErrorInfoProps
- *
- * @throws If both ensIndexerConfig and error are defined
+ * Props for ENSNodeConfigInfoView - internal component that accepts props for testing/mocking
  */
-export interface ENSNodeConfigProps {
+export interface ENSNodeConfigInfoViewProps {
```

Comment:
We probably want to rename this to `Display` so it's inline with other display pure-like components.